scaling.adoc

```diff
@@ -104,74 +104,74 @@ approaches have since been implemented.
 === Scaling approaches
 
 As hinted above, scaling Bitcoin doesn't necessarily have to be about
-increasing the block size limit or other limits. We'll go through some
-general scaling approaches, of which some don't suffer from the
+increasing the block size limit or other limits. We now go through some
+general approaches to scaling, some of which don't suffer from the
 throughput-decentralization trade-off mentioned in the previous
 section.
 
 ==== Vertical scaling
 
 Vertical scaling is the process of increasing the computing resources
-of the machines processing data. With Bitcoin, this would be
-the full node, the machine that verifies the blockchain on behalf of
-its user.
+of the machines processing data. In the context of Bitcoin, these latter would be
+the full nodes, namely the machines that validate the blockchain on behalf of
+their users.
 
-Typically, vertical scaling in Bitcoin would be an increase of the
+Typically [comm: what do you mean by "typically"? To me it suggests the idea that vertical scaling usually occurs with an increase in the block size, but I don't think this is what you mean], vertical scaling in Bitcoin would be an increase of the
 block size limit. This would require some full nodes to upgrade their
 hardware to keep up with the increasing computational
-demands. The downside is that it happens at the cost of
+demands [comm: I'm not entirely convinced by this line of reasoning; if you increase the block size limit thus the computational demand, you can either have full nodes upgrading their hardwares or just use many more of them, can't you?]. The downside is that it happens at the cost of
 centralization, as was discussed in the previous section and more
-in-depth in <<_full_node_decentralization>>.
+in depth in <<_full_node_decentralization>>.
 
 Besides the negative effects on full node decentralization, vertical
-scaling also potentially negatively impacts Bitcoin's
+scaling might also negatively impact Bitcoin's
 <<_miner_decentralization,mining decentralization>> and security in
-non-obvious ways. Let's have a look at how miners "`should`"
+less obvious ways. Let's have a look at how miners "`should`"
 operate. Say a miner mines a block at height 7 and publishes that
-block to the Bitcoin network. It will take some time for this block to
-reach broad acceptance mainly due to two factors:
+block on the Bitcoin network. It will take some time for this block to
+reach broad acceptance, which is mainly due to two factors:
 
 * Transfer of the block between peers takes time due to bandwidth
   limitations.
-* Verification of the block takes time.
+* Validation of the block takes time.
```

Comment:
Here we go! Feeling totally overridden by a ruthless majority. I'm slowly getting over this defeat.

Reply:
Ahahah it doesn't have to be a defeat! I'm honestly not so sure I understand the practical difference between verification and validation. The last time this came up, it seemed to me that @adamjonas was suggesting to use "validation" only, so that's why I changed it in the sentence. However, if you want to keep the distinction just let me know and I'll change it back.

Reply:
Don't drag me into this!

Reply:
Validation is fine. I was just fooling around.